### DISCUSSION TITLE
Preserving original domain in browser bar instead of being redirected and showing target domain

### DISCUSSION THREAD

**Discussion by suan** [chimurai/http-proxy-middleware#532] [Q&A]
I have a very straightforward setup just wanting to transparently proxy `https://mydomain.com/blog/pricing` to `https://otherdomain.com/pricing`, but have the browser bar show `mydomain.com`, NOT `otherdomain.com` – whatever I try I can't seem to get it to work, and just get redirected instead.

What am I missing?

```
const proxie = createProxyMiddleware({
  target: 'https://otherdomain.com',
  changeOrigin: true,
  secure: false,
  pathRewrite: {'^/blog' : ''},
});

app.use('/blog', proxie);
```

**Comment by SilverRat** [comment]
I set "changeOrigin: false" and that seemed to solve that problem for me.
